docs/releasehistory.rst

```diff
@@ -33,18 +47,18 @@ Behavior changed
 """"""""""""""""
 - `PR #469 <https://github.com/openforcefield/openforcefield/pull/469>`_:
   When running :py:meth:`Topology.to_openmm <openforcefield.topology.Topology.to_openmm>`, unique atom names
-  are generated (overriding any existing atom names) if the provided atom names are not unique. This
+  are generated if the provided atom names are not unique (overriding any existing atom names). This
```

Comment:
Changes to `releasehistory` below here are small grammar/formatting/syntax fixes and are not related to this PR.